src/corehost/cli/fxr/fx_muxer.cpp

```diff
@@ -1055,7 +1073,20 @@ int fx_muxer_t::execute(const int argc, const pal::char_t* argv[])
     new_argv[1] = sdk_dotnet.c_str();
 
     trace::verbose(_X("Using dotnet SDK dll=[%s]"), sdk_dotnet.c_str());
-    return parse_args_and_execute(own_dir, own_dll, 1, new_argv.size(), new_argv.data(), false, host_mode_t::muxer, &is_an_app);
+    result = parse_args_and_execute(own_dir, own_dll, 1, new_argv.size(), new_argv.data(), false, host_mode_t::muxer, &is_an_app);
+
+    //Add host usage and info
+    if (pal::strcasecmp(_X("--help"), argv[1]) == 0 ||
+        pal::strcasecmp(_X("-h"), argv[1]) == 0)
+    {
+        muxer_usage(!is_sdk_dir_present(own_dir));
```

Comment:
Shouldn't this be passing true since we just invoked the SDK in the earlier statement?